### ISSUE TITLE
Can not get the hack to install

### ISSUE THREAD

**Issue by leefur** [boxeehacks/boxeehack#564]
Hi and thanks in advance for any help…

I've tried this a variety of ways (I've probably tried installing about 30 times so far). The most nuclear option so far was:
1. Factory Restore the Boxee Box
2. Restore from USB using the image found on https://github.com/boxeehacks/boxeehack (http://dl.boxee.tv/version/dlink.dsm380/1.5.1.23735/boxee.iso)
3. Go through that initial setup wizard to get to the Settings screens and enable Windows file sharing in Network > Servers
4. Change the Host Name to boxeebox;sh /media/BOXEE/install/install.sh – with both the install and hack folders copied to the root drive of my USB stick named BOXEE

The Box's logo turns red, it resets, the host name changes to /data/hack/boot.sh. Sometimes Windows File Sharing unchecks itself so I re-check it and reboot (sometimes by backing out and shutting down, sometimes with a hard reset).

I also saw in the video that he changed the path in Workgroup, not in Host Name. So I tried that. It seemed to run the hack, but again it didn't take.

I tried formatting my USB stick with FAT and with Mac OS Extended (Journaled). 

Any ideas? Thanks so much…

**Comment by jampeef** [comment]
try to powercycle reboot the boxee. Just hold down the button on the box until it turns off then press it again for it to boot

**Comment by leefur** [comment]
Hi jampeef. Thanks. You mean after the hack is installed? Just tried that (did it twice for good measure). No change.

Does it matter how long after it turns off before I turn it back on? 

If the logo turned red and on reboot the path in Host Name changed, does that mean the hack ran successfully? Like, the USB stick I'm using is okay and in an okay format?

Grasping at straws, doing my best…

**Comment by jampeef** [comment]
it will turn red either way when you change the hostname so.  Did you copy the contents of the install folder (not the folder itself) to the root of your BOXEE usb stick? 

after the hack has been installed and the boxee has rebooted, you need to go into settings > network > servers for it to activate

**Comment by leefur** [comment]
Yeah, I had the contents of the install folder in the root of my "BOXEE" USB stick for most of my attempts. I also tried copying over the "install" and "hack" folders and adding "/install" into the path in the host name (to try running the hack locally, from what I understand).

And I went into settings > network > servers after the reboot, and then rebooted again (power cycle with the physical button and also via "shutdown").

**Comment by jampeef** [comment]
think you might need a factory reset, and then install the hack again

**Comment by leefur** [comment]
I did that (see my OP, # 1). Hmph.

**Comment by jampeef** [comment]
ok i see you wrote > boxeebox;sh /media/BOXEE/install/install.sh  ... should be: boxeebox;sh /media/BOXEE/install.sh

**Comment by leefur** [comment]
I tried that too (with the install files in the root on BOXEE). I included the extra /install when I was trying it all locally, with two folders in the root on BOXEE ("install" and "hack"). Crazy, right?

**Comment by jampeef** [comment]
weird. i'm not sure what to do.

**Comment by tbaumann** [comment]
> > after the hack has been installed and the boxee has rebooted, you need to go into settings > network > servers for it to activate

Every time? I thought it might do that at bootup?
I also just installed the hack, the logo turned red and seemingly finished installing.
But later no changes could be seen in the UI. But the hostname has changed and put it's hack hook in there.

I had the right version and I factory reseted it. I can't see what I did wrong. Except that I may be missing a step.

**Comment by MonoTovarisj** [comment]
Did you reactivate windows sharing? When you has reactivated windows sharing and the it has the boot.sh script in hostname it boots up with the plus script everytime.

**Comment by tbaumann** [comment]
Thanks. Somehow I did not realise the significance of that check box.

**Comment by pianoman235** [comment]
i have the same problem as leefur and its token it before just not any more....

**Comment by leefur** [comment]
Whoa, this is alive!?

**Comment by MonoTovarisj** [comment]
Whoaaa, somebody still using boxee? Boxee died years ago and I have long time ago burned that little shitty black box, who the developers left to die with all their users.
